frontend/src/components/common/ProductDisplay/ProductList/Product/Product.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Product.scss';
+
+interface IProp {
+  isLoading: boolean;
+  id: number;
+  name: string;
+  price: number;
+  imageUrl: string;
+}
+
+function Product(props: IProp) {
```

Comment:
name this interface `ProductProps`. it is better and easier to read.Also can be reused if you need it somewhere else, just export it